src/icalendar/tests/test_issue_335_identify_UTC.py

```diff
@@ -0,0 +1,13 @@
+
+import pytz
+import pytest
+from dateutil import tz
+from datetime import datetime
+
+try:
+    import zoneinfo
+except ModuleNotFoundError:
+    from backports import zoneinfo
+
+from icalendar import Event
```

Comment:
WIP, this is empty.

Reply:
I think I forgot to remove this file, when I moved test 335 to test_parsing. I will remove it